types/ramda/index.d.ts

```diff
@@ -1434,10 +1434,14 @@ export function none<T>(fn: (a: T) => boolean): (list: readonly T[]) => boolean;
 export function not(value: any): boolean;
 
 /**
- * Returns the nth element in a list.
- */
-export function nth<T>(n: number, list: readonly T[]): T | undefined;
-export function nth(n: number): <T>(list: readonly T[]) => T | undefined;
+ * Returns the nth element of the given list or string
+ */
+ export function nth<T>(n: number, list: readonly T[]): T | undefined;
+ export function nth(index: number, list: string): string;
```

Comment:
LGTM.
One trivial request, can you change `index` to `n`, or change all `n` to `index`?